clientv3/example_lease_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clientv3_test
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/lease"
+)
+
+func ExampleLease_create() {
+	cli, err := clientv3.New(clientv3.Config{
+		Endpoints:   endpoints,
+		DialTimeout: dialTimeout,
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer cli.Close()
+
+	kvc := clientv3.NewKV(cli)
+	lapi := clientv3.NewLease(cli)
+	defer lapi.Close()
+
+	resp, err := lapi.Create(context.TODO(), 5) // minimum lease TTL is 5-second
+	if err != nil {
+		log.Fatalf("failed to create lease %v", err)
+	}
+
+	_, err = kvc.Put(context.TODO(), "foo", "bar", clientv3.WithLease(lease.LeaseID(resp.ID)))
+	if err != nil {
+		log.Fatalf("failed to create key with lease %v", err)
+	}
+
+	time.Sleep(time.Duration(resp.TTL)*time.Second + 3*time.Second)
```

Comment:
remove this sleep and get? this is not a test, just an example to show people how to use the client to create a lease.